schema/verifiable-credential-schema.json

```diff
@@ -0,0 +1,244 @@
+{
+  "$id": "https://www.w3.org/2022/credentials/v2/json-schema.json",
```

Comment:
uncertain what to put here

Reply:
This is fine for now, the WG needs to decide what they want the new base URL to be.